### PR TITLE
v1.5.29

### PR DESCRIPTION
**Pull request by ELundby45**
# fastcore v1.5.29

## Changes
- Fork from CF
- Bump version and SHA
- Remove noarch add python deps

## Notes
- Needed for [neuralforecast](https://anaconda.atlassian.net/browse/PKG-1672?atlOrigin=eyJpIjoiMjUxYzA2ZjI0ZjRhNGIyNmJmOTg1MjFmNGVlNTdmODEiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)

### CODE REVIEW

**Review by akabanovs — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,21 +8,22 @@ package:
...
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
+    - setuptools
+    - wheel
   run:
     - packaging
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```
I don't see fastcore importing packaging. Same as pip, this should be removed.

Comment (ELundby45) [reply recipe/meta.yaml]: > I don't see fastcore importing packaging. Same as pip, this should be removed.

`pip check` fails without packaging. It is specified as a dependency in the upstream, is it preferred to override that? Also, we are installing with pip, shouldn't pip be a host dependency??

Comment (skupr-anaconda) [reply recipe/meta.yaml]: `pip` and `packaging` are runtime dependencies https://github.com/fastai/fastcore/blob/889d407805e1931d480b83174a057179abbdc7ea/setup.py#L29

Comment (cbouss) [reply recipe/meta.yaml]: This looks to me to be an error from upstream. I think we should patch setup.py to remove runtime dependencies on pip and packaging. (and set pip and packaging in host).

Comment (skupr-anaconda) [reply recipe/meta.yaml]: I agree they seem to be not run dependencies and there are no mentions in the source code

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**